### ISSUE TITLE
typing in numboxes while [display] is updating

### ISSUE THREAD

**Issue by tomara-x** [plugdata-team/plugdata#1733]
```
#X obj 429 149 display;
#X obj 429 79 lfo 89;
#X connect 1 0 0 0;
```

symbols/message boxes are not affected, only numbox/floatbox

https://github.com/plugdata-team/plugdata/assets/86204514/ad43c650-5c60-4101-b2f9-00a3f602e9cd

**Comment by alcomposer** [comment]
This is the same behavior as Vanilla.

https://github.com/user-attachments/assets/8a69bd39-9a52-4e7b-8733-da28c6ebfbbf

**Comment by alcomposer** [comment]
This looks like it's due to canvas message `coords` in the [display] object:
![image](https://github.com/user-attachments/assets/6e0b033c-5d18-4ebc-bfc7-1cc5b80656d1)
```
#X obj 174 249 namecanvas myCanvas;
#X obj 175 279 lfo 1;
#X obj 277 281 nbx 4 18 -1e+37 1e+37 0 0 empty empty empty 0 -8 0 10 #191919 #e1e1e1 #e1e1e1 0 256;
#X msg 175 314 \; myCanvas coords 0 1 1 -1 0 22 2 100 100 1, f 43;
#X connect 1 0 3 0;
```

**Comment by tomara-x** [comment]
cool, thanks! i'll give this a test later

**Comment by tomara-x** [comment]
it does indeed fix this, great work!

you can add `closes #1733`  to the pr description to make it clearer
